test/js/sql/sqlite-sql.test.ts

```diff
@@ -1311,6 +1311,130 @@ describe("SQL helpers", () => {
     expect(results[0].value).toBe("test");
   });
 
+  test("insert select helper with IN", async () => {
```

Comment:
I'm not 100% sure what this test is testing, would you mind giving me a little bit of context?